.gitlab-ci-check-golang-lint.yml

```diff
@@ -40,3 +40,18 @@ test:static:
     - fi
   script:
     - golangci-lint run -v
+
+test:govendor-check:
+  stage: test
+  needs: []
+  variables:
+    - GOLANG_VERSION=1.16
```

Comment:
Keeping this open for multiple versions in different repos.

Reply:
Okey, but shouldn't the default be today's latest?

Reply:
Perhaps, I just aligned with the already existing check here, which uses 1.16.
Another point is that we know different versions of the toolchain acts a little different, so should be aligned to the repo's version most likely (was my thining at least).
But I see your point with master, and thinking bout it, it's probably best :thinking:

Reply:
It's okey, you have a fair point too. Let's lock it to 1.16 for now.